week3/Week3HomeWork/Controllers/ConverterController.cs

```diff
@@ -0,0 +1,34 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using Week3HomeWork.Models;
+
+namespace Week3HomeWork.Controllers
+{
+    [ApiController]
+    [Route("[controller]")]
+    public class ConverterController : ControllerBase
+    {
+        [HttpGet("GallonsToLitres")]
+        public double GallonsToLitres(int Litres)
```

Comment:
Name of variables and parameters should start with lowercase

Reply:
Done